config/initializers/inflections.rb

```diff
@@ -4,13 +4,13 @@
 # are locale specific, and you may define rules for as many different
```

Comment:
Same comment as above. Does this file need to be checked in if we don't uncomment anything?

Reply:
yeah, good to have this file here in the unlikely event we do need it. (We've found a lot of benefit in keeping the state of this kinda thing as close to the current rails state of the art as possible, so that if we do have to start using something we don't have to figure out what the initial shape should be.)